ui/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Maintenance tracker</title>
+  <link href="style.css" rel="stylesheet">
+  <!--google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Skranji" rel="stylesheet">
+  <!--font awesome -->
+  <script src="fontawesome-all.js"></script>
+</head>
+
+<body>
+  <!--header-section-begin-->
+  <header class="header">
+    <nav>
+      <div class="nav-container">
+        <div class="btn">
+          <div class="bar"></div>
+          <div class="bar"></div>
+          <div class="bar"></div>
+        </div>
+      </div>
+      <ul>
+        <div class="logo">Maintenance Tracker</div>
+        <li>
+          <a href="#">home</a>
+        </li>
+        <li>
+          <a href="#about-section">about</a>
+        </li>
+        <li>
+          <a href="signin.html">sign in</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="banner">
+      <div class="banner-center">
+        <h1>Maintenance Tracker</h1>
+        <h3>Repairing one day at a time</h3>
+        <div>
+          <a class="button" href="signin.html">Sign In</a>
```

Comment:
These buttons (Signin and Signup) could be made larger for the bigger screen.

Reply:
Okay, I will fix it